src/IoC/ServiceExtensions.cs

```diff
@@ -130,7 +131,10 @@ public static IServiceCollection AddBuilders(this IServiceCollection services)
                 .AddScoped<IBuilder<Workstation>, WorkstationResourceBuilder>()
                 .AddScoped<IBuilder<WorkstationElement>, WorkstationElementsResourceBuilder>()
                 .AddScoped<IBuilder<PcasStorageType>, PcasStorageTypeResourceBuilder>()
-                .AddScoped<IBuilder<PcasBoard>, PcasBoardResourceBuilder>();
+                .AddScoped<IBuilder<PcasBoard>, PcasBoardResourceBuilder>()
+                .AddScoped<IBuilder<StoresPallet>, StoresPalletResourceBuilder>()
+                .AddScoped<IBuilder<StorageLocation>, StorageLocationResourceBuilder>()
+                .AddScoped<IBuilder<LocationType>, LocationTypeResourceBuilder>();
```

Comment:
you've registered all these as IoC dependencies so might as well inject them in the StoresPalletResourceBuilder (like I said before)